### PR TITLE
Remove custom brew

### PR DESCRIPTION
**Pull request by dominikschulz**
Gopass has an official homebrew formula. Remove our custom one.

RELEASE_NOTES=[CLEANUP] Remove the custom formula in favor of the
official one.

Signed-off-by: Dominik Schulz <dominik.schulz@gauner.org>

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -139,14 +139,15 @@ brew install pinentry-mac
...
 ```
 
-### Ubuntu & Debian
+### Ubuntu, Debian, Deepin, Devuan, Kali Linux, Pardus, Parrot, Raspbian
 
-**WARNING**: The official Debian repositories contain a package named `gopass` that
-is not related to this project in any way. It's a similar tool with a completely
-independent implementation and feature set. We are aware of this issue but can not
-do anything about it.
+**WARNING**: The official Debian repositories (and dervied distributions) contain
```
Comment (AnomalRoil) [inline docs/setup.md:144]: derived

File: docs/setup.md [hunk 2]
```diff
@@ -139,14 +139,15 @@ brew install pinentry-mac
...
+a package named `gopass` that is not related to this project in any way.
+It's a similar tool with a completely independent implementation and feature set.
+We are aware of this issue but can not do anything about it.
 
-When installing on Ubuntu or Debian you can either download the `deb` package and install manually or use our repository.
+When installing on Ubuntu or Debian you can either download the `deb` package
+and install manually or build from source.
```
Comment (AnomalRoil) [inline docs/setup.md:150]: I'd say the recommended way should be to `go get` gopass, no ?

**Review by AnomalRoil — CHANGES_REQUESTED** [review]

File: docs/setup.md (hunk 1 above)
Comment (dominikschulz) [reply docs/setup.md:144]: Thanks, I should really remember to run aspell ...

File: docs/setup.md (hunk 2 above)
Comment (dominikschulz) [reply docs/setup.md:150]: Yes, that's what I mean by "or build from source". How would you change that sentence?

File: docs/setup.md (hunk 1 above)
Comment (AnomalRoil) [reply docs/setup.md:144]: Make it a git pre-commit hook :D

File: docs/setup.md (hunk 2 above)
Comment (AnomalRoil) [reply docs/setup.md:150]: Make it an anchor link maybe?
`[install manually or build from source](#installing-from-source)`

Comment (dominikschulz) [reply docs/setup.md]: sounds good

**Review by AnomalRoil — APPROVED** [review]

**Pull request merged**